Tests/CarthageKitTests/ProjectSpec.swift

```diff
@@ -406,6 +406,17 @@ class ProjectSpec: QuickSpec {
 				expect(events) == [.downloadingBinaryFrameworkDefinition(.binary(testDefinitionURL), testDefinitionURL)]
 			}
 		}
+		
+		describe("platformForFramework") {
+			let testStaticFrameworkURL = Bundle(for: type(of: self)).url(forResource: "Alamofire.framework", withExtension: nil)!
+			
+			it("should produce a platform") {
```

Comment:
Can you specify whether this is checking the binary or the info plist?

If it's not checking the binary, we should add a test for that.

Reply:
It's checking the Binary ;) The .plist is not even present in the bundle.